.gitignore

```diff
@@ -6,6 +6,10 @@ __pycache__/
 # C extensions
 *.so
 
+# Files
+Docs/
+langchain/
```

Comment:
Why do we need this in .gitignore?

Reply:
docs contain the local files 
langchain contain the virtual environment files 
To avoid adding them by mistake , added them to .gitignore